clients/feeder/feeder.go

```diff
@@ -142,7 +145,7 @@ func newTestServer(network utils.Network) *httptest.Server {
 	}))
 }
 
-func NewClient(clientURL string) *Client {
+func NewClient(clientURL, version string) *Client {
```

Comment:
Can we have a `WithUserAgent(ua string)` function instead of passing a version in here? That way it's a non-breaking change and more generic.

Reply:
Agreed